darts-api-stub/transcriptions/transcriptions.js

```diff
@@ -26,6 +40,20 @@ router.get('/urgencies', (req, res) => {
   ]);
 });
 
+router.get('/:transcriptId', (req, res) => {
+  switch (req.params.transcriptId) {
+    case '1':
+      res.status(200).send(mockTranscriptionDetails);
+    default:
+      const resBody104 = {
+        type: 'CASE_104',
```

Comment:
The error from the API would be
```suggestion
        type: 'TRANSCRIPTION_101',
```

Reply:
This change will be in the incoming commit